src/index.html

```diff
@@ -388,6 +388,23 @@ <h1>COVID-19 Spread Simulator 🦠</h1>
               </select>
             </span>
           </div>
+          <div class="parameterField">
+            <div class="parameterTextLine tooltip ">
+              Conformists (%)
```

Comment:
As suggested by @marcopierobon on slack, it would be better to replace "Conformists" with "Self-isolating"

Reply:
Done